pcsx2-qt/QtKeyCodes.cpp

```diff
@@ -13,6 +13,53 @@
 
 #include <QtGui/QKeyEvent>
 
+uint8_t map_text_to_keycode(const QString& text)
```

Comment:
Better to use u8 for consistency, it's the same.

Reply:
understood, made that change